types/less/index.d.ts

```diff
@@ -1,18 +1,29 @@
-// Type definitions for LESS
+// Type definitions for LESS 3.x
 // Project: http://lesscss.org/
 // Definitions by: Tom Hasner <https://github.com/thasner>
 //                 Pranay Prakash <https://github.com/pranaygp>
 //                 Daniel Waxweiler <https://github.com/dwaxweiler>
+//                 Richard Lea <https://github.com/chigix>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 declare namespace Less {
+    // https://github.com/less/less.js/blob/master/lib/less/import-manager.js#L10
     interface RootFileInfo {
+        /** whether to adjust URL's to be relative */
+        rewriteUrls?: boolean;
+        /** full resolved filename of current file */
         filename: string;
         relativeUrls: boolean;
+        /** path to append to normal URLs for this node */
         rootpath: string;
+        /** path to the current file, absolute */
         currentDirectory: string;
+        /** absolute path to the entry file */
         entryPath: string;
+        /** filename of the base file */
         rootFilename: string;
+        /** whether the file should not be output and only output parts that are referenced */
+        reference: boolean;
```

Comment:
Where is this attribute mentioned?

Reply:
Mentioned in the document comment in official repository: 
https://github.com/less/less.js/blob/master/lib/less/import-manager.js#L17

Reply:
Thanks!